### PR TITLE
Documentation update 

### PR DESCRIPTION
**Pull request by Volland**
- clarify encrypted seed - btw it is nice to have a Glossary section for terms and definitions
- add revocation flow
- minor update - add missed params to passwordless login

### CODE REVIEW

File: sdk/core/README.md [hunk 1]
```diff
@@ -37,6 +38,10 @@
     - [Validate Holder Response on Offer Request](#validate-holder-response-on-offer-request)
     - [Sign multiple credentials](#sign-multiple-credentials)
     - [Generate signed credential](#generate-signed-credential)
+  - [Revocation](#revocation)
+    - [Flow](#Revocation Flow)
+    - [Issuance](#issuance of Revocable credential)
+    - [Revocation](#Revocation of Revocable credential)
```
Comment (launganik) [inline sdk/core/README.md:44]: @Volland looks like this format is incorrect. I think you should have a `*` before [Revocation] 
Right now it looks like this:
image

File: sdk/core/README.md [hunk 2]
```diff
@@ -96,6 +101,8 @@ const options = {
   accessApiKey: 'YOUR API KEY HASH VALUE'
 }
 ```
+`encryptedSeed`- randomly auto generated by [Generate seed](#Generate seed) [more detail exaple](#Initialize from seed) material that used as input for a private key generation. Generated as a part of [register](#Create DID) / [signUp](#Sign up) process and stored as encrypted value in protected vault.
```
Comment (launganik) [inline sdk/core/README.md:104]: @Volland same here, the links do not display correctly

Comment (launganik) [reply sdk/core/README.md:104]: typo - `more detailed example`

File: sdk/core/README.md [hunk 3]
```diff
@@ -702,6 +710,53 @@ const credential = await issuer.signCredential(
 `credentialMetadata` - schema of credential (should be defined and Issuer and Verifier use the same,
 so verifier will be able to understand what kind of credential was created by Issuer).
 
+### Revocation
+SDK Support Issuing Revocable Credential based on [Revocation List 2020 W3C standard](https://w3c-ccg.github.io/vc-status-rl-2020)
+- [Concepts](https://w3c-ccg.github.io/vc-status-rl-2020/#core-concept)
+- [revocationlist2020credential](https://w3c-ccg.github.io/vc-status-rl-2020/#revocationlist2020credential)
+- [revocationlist2020status](https://w3c-ccg.github.io/vc-status-rl-2020/#revocationlist2020status)
+
+####  Revocation Flow
```
Comment (launganik) [inline sdk/core/README.md:719]: Delete this line

**Review by launganik — CHANGES_REQUESTED** [review]
@Volland added some comments for changing before merge

File: sdk/core/README.md [hunk 4]
```diff
@@ -96,6 +101,8 @@ const options = {
   accessApiKey: 'YOUR API KEY HASH VALUE'
 }
 ```
+`encryptedSeed`- randomly auto generated by [Generate seed](#Generate seed) [more detail example](#initialize-from-seed) material that used as input for a private key generation. Generated as a part of [register](#create-did) / [signUp](#sign-up) process and stored as encrypted value in protected vault.
```
Comment (launganik) [inline sdk/core/README.md:104]: @Volland generate-seed link is still not correct

**Review by launganik — APPROVED** [review]
@Volland one more change suggested. Then it's good to go :)

**Review by launganik — APPROVED** [review]

**Review by rbrazhnyk — APPROVED** [review]

**Pull request merged**